src/Form/AutologoutSettingsForm.php

```diff
@@ -167,36 +199,35 @@ public function buildForm(array $form, FormStateInterface $form_state) {
    * {@inheritdoc}
    */
   public function validateForm(array &$form, FormStateInterface $form_state) {
-    $input_values = $form_state->getUserInput();
-    $max_timeout = $input_values['autologout_max_timeout'];
+    $max_timeout = $form_state->getValue('autologout_max_timeout');
```

Comment:
Instead of getting the values individually every time, use `$form_state->getValues();` to get all the values in a array and use the array to check. This is done correctly in the `submitForm()` function.